llvm/lib/IR/Core.cpp

```diff
@@ -2893,6 +2894,34 @@ LLVMBool LLVMHasNoSignedWrap(LLVMValueRef Inst) {
   return false;
 }
 
+int LLVMGetSignInfo(LLVMValueRef Inst){
+  DbgDeclareInst* dbg_declare_inst = dyn_cast<DbgDeclareInst>(unwrap(Inst));
```

Comment:
The naming style of variable names in this function isn't consistent with other functions in this file.
@thanhtoantnt: can you refactor the code?